src/Model/Behavior/GeocoderBehavior.php

```diff
@@ -42,7 +43,9 @@ class GeocoderBehavior extends Behavior {
 		'address' => null,
 		'allowEmpty' => true,
 		'expect' => [],
-		'lat' => 'lat', 'lng' => 'lng', 'formatted_address' => 'formatted_address',
+		'lat' => 'lat', 'lng' => 'lng',
+		'formatted_address' => 'formatted_address',
+		'address_format' => '%S %n, %z %L', // For class StringFormatter
```

Comment:
shouldnt this be camelBacked as per convention?

Reply:
formatted_address is also not clean, though, I just saw ;)

Reply:
I dunno ¯\_(ツ)_/¯
I used snake_case as that's what was there already. But I might have used it either way. I convert to-and-from json alot, so I tend to avoid UpperCase keys.

Reply:
You can add the new one as `addressFormat`
I will fix the other one in a BC one afterwards :)

Reply:
👍